apps/console/src/features/userstores/constants/user-store-constants.ts

```diff
@@ -177,3 +177,8 @@ export const CONSUMER_USERSTORE: string = "DEFAULT";
  * Name of the primary userstore.
  */
 export const PRIMARY_USERSTORE: string = "PRIMARY";
+
+/**
+ * Character limit for userstore name.
+ */
+export const USERSTORE_NAME_CHARACTER_LIMIT: number = 50;
```

Comment:
AFAIK, user store name can have a length of upto 255 chars.

Reply:
could you add your thoughts with a reference on the thread sent to you.

Reply:
I could enter a userstore name with 251 characters and proceed with userstore creation without any issue. The reasoning for my suggestion was that if there is a requirement to use a long userstore name for some reason, 50 char limit can be a blocker for that.

Recently too we had to revise the username lengths following a customer request to allow long usernames. Was just wondering whether a similar case would arise in the future for userstore creation as well.

Reply:
Since a domain name of a user store is meant to be simple and easy to remember IMO it would be alright to go with a max character length of 50.